PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/DynamicThumbstick.rbxmx

```diff
@@ -88,7 +88,7 @@ function Thumbstick:Enable()
 	local humanoid = MasterControl:GetHumanoid()
 	if humanoid then
 		local shouldRevert = humanoid.AutoJumpEnabled == false
-		shouldRevert = shouldRevert and humanoid.DevTouchMovementMode == Enum.DevTouchMovementMode.UserChoice
```

Comment:
did this not exist on humanoid? or is that just the wrong level to be checking at?